5.6/apache/Dockerfile

```diff
@@ -50,5 +50,5 @@ RUN { \
     && a2enmod rewrite
 
 # Install composer
-COPY install-composer /install-composer
+COPY image-files /
```

Comment:
@schmunk42 Actually I think, we should try to remove the install script completely and instead convert it to 2-3 lines in the `Dockerfile`. The instructions for how to install composer on getcomposer.com are confusing now to put it mildly. They show 2 ways, both of them using weird code. We can do better I think. For example this code from their website:

```
EXPECTED_SIGNATURE=$(wget -q -O - https://composer.github.io/installer.sig)
php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
```

They use `wget` in the first line and php in the second line for one and the same task: download a file. Why???

I can provide a simplified set of commands, if you want.